### PR TITLE
pacman-mirrors: add page

### PR DESCRIPTION
**Pull request by marchersimon**
The man page and the Manjaro wiki state that it's necessary to run `sudo pacman -Syyu` after every run of `pacman-mirrors`. However I'm not sure if and where I should add that to the tldr page.

### CODE REVIEW

File: pages/linux/pacman-mirrors.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+> Generate a pacman mirrorlist for Manjaro Linux.
+> More information: <https://wiki.manjaro.org/index.php?title=Pacman-mirrors>.
+
+- Generate a mirrorlist, using the default settings:
+
+`sudo pacman-mirrors -f`
+
+- Get the status of the current mirrors:
+
+`sudo pacman-mirrors --status`
```
Comment (navarroaxel) [inline pages/linux/pacman-mirrors.md:12]: Doesn't require sudo.
```suggestion
`pacman-mirrors --status`
```

File: pages/linux/pacman-mirrors.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# pacman-mirrors
+
+> Generate a pacman mirrorlist for Manjaro Linux.
+> More information: <https://wiki.manjaro.org/index.php?title=Pacman-mirrors>.
+
+- Generate a mirrorlist, using the default settings:
+
+`sudo pacman-mirrors -f`
```
Comment (navarroaxel) [inline pages/linux/pacman-mirrors.md:8]: We prefer long arguments
```suggestion
`sudo pacman-mirrors --fasttrack`
```

File: pages/linux/pacman-mirrors.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Get the status of the current mirrors:
+
+`sudo pacman-mirrors --status`
+
+- Check, which branch you are on:
+
+`sudo pacman-mirrors --get-branch`
```
Comment (navarroaxel) [inline pages/linux/pacman-mirrors.md:16]: Doesn't require sudo.
```suggestion
`pacman-mirrors --get-branch`
```

File: pages/linux/pacman-mirrors.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Generate a mirrorlist, using the default settings:
+
+`sudo pacman-mirrors -f`
+
+- Get the status of the current mirrors:
+
+`sudo pacman-mirrors --status`
+
+- Check, which branch you are on:
```
Comment (navarroaxel) [inline pages/linux/pacman-mirrors.md:14]:
```suggestion
- Display which branch you are on:
```

File: pages/linux/pacman-mirrors.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`sudo pacman-mirrors --get-branch`
+
+- Switch to a different branch:
+
+`sudo pacman-mirrors --api --setbranch {{stable|unstable|testing}}`
```
Comment (navarroaxel) [inline pages/linux/pacman-mirrors.md:20]: A typo here:
```suggestion
`sudo pacman-mirrors --api --set-branch {{stable|unstable|testing}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/linux/pacman-mirrors.md (hunk 4 above)
Comment (bl-ue) [reply pages/linux/pacman-mirrors.md:14]: I think this is better:
```suggestion
- Display the current branch:
```

Comment (navarroaxel) [reply pages/linux/pacman-mirrors.md:14]: Thanks @bl-ue !

File: pages/linux/pacman-mirrors.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# pacman-mirrors
+
+> Generate a pacman mirrorlist for Manjaro Linux.
```
Comment (navarroaxel) [inline pages/linux/pacman-mirrors.md:3]:
```suggestion
> Generate a pacman mirrorlist for Manjaro Linux.
> Every run of pacman-mirrors requires you to synchronize your database and update your system using `sudo pacman -Syyu`.
```

**Review by navarroaxel — COMMENTED** [review]
I think we can add the warning in the command description section.

**Review by bl-ue — APPROVED** [review]

File: pages/linux/pacman-mirrors.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+> Every run of pacman-mirrors requires you to synchronize your database and update your system using `sudo pacman -Syyu`.
+> More information: <https://wiki.manjaro.org/index.php?title=Pacman-mirrors>.
+
+- Generate a mirrorlist, using the default settings:
```
Comment (bl-ue) [inline pages/linux/pacman-mirrors.md:7]: Whoops! found one thing:
```suggestion
- Generate a mirrorlist using the default settings:
```

**Review by bl-ue — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**